Source/WebInspectorUI/UserInterface/Views/Variables.css

```diff
@@ -424,7 +424,7 @@ body {
         }
     }
     
-    &:not(.mac-platform.monterey, .mac-platform.big-sur):not(.docked) {
+    &:is(.mac-platform):not(.mac-platform.monterey, .mac-platform.big-sur):not(.docked) {
```

Comment:
i feel like this would be better written as
```css
&:is(.mac-platform):not(.monterey, .big-sur):not(.docked) {
```
or even more ideally as
```css
&:is(.mac-platform.catalina):not(.docked) {
```
since i dont think we support anything before macOS 10.15 Catalina

Reply:
@dcrousso I'll send a follow-up, for this I tried to apply only the smallest change that would avoid applying the non-zero size to non-Mac platforms.

Reply:
> @dcrousso I'll send a follow-up, for this I tried to apply only the smallest change that would avoid applying the non-zero size to non-Mac platforms.

Done, PR at https://github.com/WebKit/WebKit/pull/28287

I saw still references to Mojave in the code, so I kept it in the selector.